### PR TITLE
Add Rain.bh to exchanges

### PR DESCRIPTION
**Pull request by champbronc2**
Requesting to add our Middle Eastern platform, [Rain.bh](https://rain.bh).

Rain officially launched and received licensure from the Central Bank of Bahrain on [July 31, 2019](https://www.coindesk.com/crypto-exchange-rain-raises-from-bitmex-opens-trading-in-middle-east). We natively support: BHD, KWD, AED, SAR, OMR, and USD. Open for signup from all jurisdictions besides US and sanctioned countries.

### CODE REVIEW

File: _templates/exchanges.html [hunk 1]
```diff
@@ -108,6 +140,16 @@ <h3 id="south-korea" class="no_toc">South Korea</h3>
       </div>
     </div>
 
+    <div class="row marketplace">
+      <img class="marketplace-flag" src="/img/flags/SA.png?{{site.time | date: '%s'}}" alt="Saudi Arabia flag">
```
Comment (wbnns) [inline _templates/exchanges.html]: Please include an SVG instead of a PNG.

File: _templates/exchanges.html [hunk 2]
```diff
@@ -84,6 +106,16 @@ <h3 id="malaysia" class="no_toc">Malaysia</h3>
       </div>
     </div>
 
+    <div class="row marketplace">
+      <img class="marketplace-flag" src="/img/flags/OM.png?{{site.time | date: '%s'}}" alt="Oman flag">
```
Comment (wbnns) [inline _templates/exchanges.html]: Please include an SVG instead of a PNG.

File: _templates/exchanges.html [hunk 3]
```diff
@@ -74,6 +86,16 @@ <h3 id="japan" class="no_toc">Japan</h3>
       </div>
     </div>
 
+    <div class="row marketplace">
+      <img class="marketplace-flag" src="/img/flags/KW.png?{{site.time | date: '%s'}}" alt="Kuwait flag">
```
Comment (wbnns) [inline _templates/exchanges.html]: Please include an SVG instead of a PNG.

File: _templates/exchanges.html [hunk 4]
```diff
@@ -40,6 +42,16 @@ <h2 class="exchanges-tab-title exchanges-p2p-title" id="p2p">Peer-to-Peer (P2P)<
   <h2 class="exchanges-tab-title exchanges-asia-title" id="asia">Asia</h2>
   <div class="marketplace-row">
 
+    <div class="row marketplace">
+      <img class="marketplace-flag" src="/img/flags/BH.png?{{site.time | date: '%s'}}" alt="Bahrain flag">
```
Comment (wbnns) [inline _templates/exchanges.html]: Please include an SVG instead of a PNG.

File: _templates/exchanges.html [hunk 5]
```diff
@@ -18,6 +18,8 @@ <h2 class="exchanges-tab-title exchanges-international-title" id="international"
     <a class="marketplace-link" href="https://coinmama.com/">Coinmama</a>
     <br>
     <a class="marketplace-link" href="https://www.kraken.com/">Kraken</a>
+    <br>
+    <a class="marketplace-link" href="https://www.rain.bh/">Rain</a>
```
Comment (wbnns) [inline _templates/exchanges.html]: As you all are a new exchange, please revise the PR to remove the addition to the International category. We also note that you all do not appear in the list of [250+ exchanges reporting volume](https://coinmarketcap.com/rankings/exchanges/reported/) on CoinMarketCap, which would further preclude the addition to this section.

Comment (champbronc2) [reply _templates/exchanges.html]: No worries, done. We have our public orderbook API currently in development, so can just open up another PR down the road once we are listed on CMC.

Thanks for your review @wbnns , changes pushed!

**Comment by syaifudin857** [comment]
Join

**Comment by champbronc2** [comment]
@wbnns changes made, appreciate if you can re-review!

**Comment by wbnns** [comment]
@champbronc2 

Hello, we are attempting to review your exchange again, however, we're not receiving the verification code upon creation of the account. We're using a Canadian number capable of receiving text messages. We opened a support ticket with you all.

**Comment by champbronc2** [comment]
@wbnns why was the branch closed? The phone issue was due to using a VoIP, but we have bypassed that for your testing account. Appreciate your time/consideration

**Comment by wbnns** [comment]
@champbronc2 

Hello, we recommended to the support team (which they said had been passed along to engineering) that an error message should be added to the sign up, so that users know why they aren't receiving a text message in the event a VoIP text-capable number is used. 

Please let us know when this is done and we will resume the review.

**Comment by champbronc2** [comment]
@wbnns performed some investigation on this case here. We do not explicitly disallow all VoIP #'s, Seems some VoIP's function properly. It was moreseo that our SMS provider ([Authy/Twilio](https://support.authy.com/hc/en-us/articles/115001750428-Authy-Landline-VoIP-connectivity-issues)) can experience deliverability issues to VoIP #'s in specific, however their API does not report if the SMS was successfully delivered, only if it was attempted. Even after contacting them, they were not 100% certain why the SMS was not received to your #. 

I'd also say it is common for KYC/AML exchanges to not have full support for VoIP #'s. 

Really do apologize for the inconvenience your account experienced, but with the above in mind, and that your test account has had phone verification bypassed, may I request the reviewed be resumed? Should be smooth experience moving forward.

Thank you.

**Comment by champbronc2** [comment]
@wbnns Any chance you saw my above message? https://github.com/bitcoin-dot-org/bitcoin.org/pull/3108#issuecomment-545196818

**Comment by champbronc2** [comment]
@Cobra-Bitcoin we had an outstanding exchange review paid for, any chance we can have a refund, or PR approval?

The last request before approval from their feedback was adding an export transaction history feature, which we have in progress, and slated to go out shortly. But not sure if the review standards are exactly the same now.

**Comment by Cobra-Bitcoin** [comment]
If you do a new pull request with the conflicts resolved (there's some issues with this branch), I will merge it.

**Pull request closed without merging**